app/resources/nested_resources/file_metadata.rb

```diff
@@ -29,6 +29,9 @@ class FileMetadata < Valkyrie::Resource
   attribute :fixity_success, Valkyrie::Types::Integer
   attribute :fixity_last_success_date, Valkyrie::Types::DateTime.optional
 
+  # preservation attributes
+  attribute :preservation_copy_of_id, Valkyrie::Types::ID.optional
```

Comment:
Sorry I meant to come back and check on this comment again before submitting!! I'm still not sure what it means, though, and I see I was confused when it was used, also. Maybe an extra comment would help?

Reply:
👍 that clarifies it. I was wondering about this, actually. From the code it seems that once a file is preserved it will never be preserved again. Is that because of our understanding that a file can never be changed, but instead would be deleted / re-created? I have a vague memory that there is a geo metadata file that doesn't align with this assumption.